app/ui/browser/reducers/main-reducers.js

```diff
@@ -37,15 +37,26 @@ function getPageIndexById(state, id) {
 export default function basic(state = initialState, action) {
   switch (action.type) {
     case types.CREATE_TAB:
```

Comment:
types.CREATE_TAB is used in `<TabBar/>` (defined in `browser.jsx` as `handleNewTabClick`) which is the main local way to create a tab -- the context menus are still non-reduxy (and a lot of the things in `actions/external.js` for the most part) and some have the roundtrip IPC stuff, and those seem mostly legacy from the initial prototype.

I guess the main difference between the two types is distinguishing the source of the action, or what caused it, for better debugging, and possibly replayability